src/dbnode/storage/database.go

```diff
@@ -460,6 +460,9 @@ func (d *db) AssignShardSet(shardSet sharding.ShardSet) {
 	d.shardSet = shardSet
 	if receivedNewShards {
 		d.lastReceivedNewShards = d.nowFn()
+		// we need to disable file ops so that warm/cold flush is not running during shards update.
+		// bootstrap will enable file ops when it completes.
```

Comment:
Nit:
```suggestion
		// We need to disable file ops so that warm/cold flush is not running during shards update.
		// Bootstrap will enable file ops when it completes.
```